charts/oidc-proxy/templates/configmap.yaml

```diff
@@ -12,12 +12,14 @@ data:
     ServerTokens ProductOnly
     TraceEnable off
 
-    LogFormat "%h %l %u \"%{OIDC_CLAIM_email}i\" \"%{X-App-ID}i\" [%{%FT%T}t.%{msec_frac}t%{%z}t] %D %X \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\"" combined
-    LogFormat "%{X-Forwarded-For}i %l %u \"%{OIDC_CLAIM_email}i\" \"%{X-App-ID}i\" [%{%FT%T}t.%{msec_frac}t%{%z}t] %D %X \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\"" proxy
+    LogFormat "%h %l %u \"%{OAUTH2_CLAIM_email}i\" \"%{X-App-ID}i\" [%{%FT%T}t.%{msec_frac}t%{%z}t] %D %X \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\"" combined
```

Comment:
Just fyi, AppSec wanted to add `\"%{Origin}i\"` to the log line. I added it between Referer and User-Agent in the other proxies. See: https://broadworkbench.atlassian.net/browse/DDO-1983, [Sam site.conf](https://github.com/broadinstitute/firecloud-develop/blob/dev/base-configs/sam/site.conf.ctmpl#L98).

Not sure if it's needed for TDR or how your log infrastructure works. :)

Reply:
Ooh good call.  I can totally add that